Yoast/YoastSEO.js
# Add functionality to recognize German verb forms

## Summary

This PR can be summarized in the following changelog entry:

* [not-user-facing] Adds basic functionality to create German verb forms.

## Relevant technical choices:

* This PR includes basic verb functionality. This includes adding regular suffixes and generating forms for strong verbs. Extra functionality will be added in later PRs.

## Test instructions

This PR can be tested by following these steps:

* Make sure you have the premium configuration set to the right branch (see https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/10).
* Change the language of your site to German.
* Add a German text and a keyphrase including German verbs.
* Add different grammatical forms of the verbs in your text; make sure these forms are recognized. For this, you use forms of a specific word from a dictionary, e.g. https://de.wiktionary.org/wiki/Flexion:kaufen

Fixes #2132
